.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+  "root": true,
```

Comment:
I may be misunderstanding but I think `"root": true` is only necessary in nested folders, when the interest is to stop searching for eslintrc in parent directories and use the eslintrc in the root of the nested directory, reading https://eslint.org/docs/user-guide/configuring

Reply:
That is true, but there's no guarantee that someone won't check out this repo into a directory with a .eslintrc file above it.